src/components/Player/Pages/Overview/Overview.jsx

```diff
@@ -29,23 +21,34 @@ const getPlayerMatchesAndHeroes = (playerId, options) => (dispatch) => {
 };
 
 const MAX_OVERVIEW_ROWS = 20;
+const OVERVIEW_TABLES_BREAK_SIZE = 1224;
```

Comment:
Actually we no need `1224` somewhere else since exactly from this widths tables on Overview page starting to deteriorate